src/main/java/dk/cngroup/lentils/service/ImageService.java

```diff
@@ -125,4 +128,14 @@ public Image getImage(final Long imageId) {
     public Image getImageFromMultipartFile(final MultipartFile file) {
         return new Image(getFilePath(file));
     }
-}
+
+    @Transactional
```

Comment:
Ted mi neni z kodu moc jasne, proc je tady potrebna anotace Transactional?
Jestli na to neprijdu, dej prosim koment :)

Reply:
Nepřišels na to dobře, není to potřeba :-)